resources/idlharness.js

```diff
@@ -1069,13 +1069,19 @@ IdlInterface.prototype.test_self = function()
         // "The class string of an interface prototype object is the
         // concatenation of the interface’s identifier and the string
         // “Prototype”."
-        assert_class_string(self[this.name].prototype, this.name + "Prototype",
-                            "class string of " + this.name + ".prototype");
+
```

Comment:
This needs to be done in the idlharness repo, and then the git submodule updated, not done directly here.